lib/device-manager.ts

```diff
@@ -289,7 +289,7 @@ export class DeviceManager implements IDeviceManager {
                 args.device.config['offsetPixels'] = AndroidController.calculateScreenOffset(args.device.config.density);
             }
 
-            if (!density) {
+            if (!density && !args.isIOS) {
```

Comment:
We have a case when there is no density in Android, but what about in iOS? Don't we need to set the offsetPixels?